web/src/components/audio-player/AudioPlayer.vue

```diff
@@ -775,6 +757,31 @@ export default class AudioPlayer extends Vue {
       window.clearInterval(this.progressInterval);
     }
   }
+
+  setBackgroundFromImage() {
+    Vibrant.from(this.artwork)
+      .getPalette()
+      .then((palette) => {
+        const swatch = palette.DarkMuted;
+        if (!swatch) {
+          return;
+        }
+        this.vibrantBackgroundColor = swatch.getHex();
+        this.vibrantTextColor = swatch.getBodyTextColor();
+      });
+  }
+
+  scrollToCurrentLyricsGroup(id) {
+    if (id === null) {
+      window.scrollTo(0, 0);
```

Comment:
I don't understand what this is doing.

Reply:
So currently we have a track that is playing with the new lyrics format

Now when the next track starts to play and the next track is the old format, then it doesnt scroll to the top automatically, so this line is scrolling to the top of the component if we do not have any ID